src/op_mode/nat.py

```diff
@@ -277,7 +283,21 @@ def _get_formatted_translation(dict_data, nat_direction, family):
     return output
 
 
+def _verify():
+    config = ConfigTreeQuery()
+    if not config.exists(base):
+        return False
+    return True
+
+
 def show_rules(raw: bool, direction: str, family: str):
+    # check if config exist
+    if not _verify():
```

Comment:
I believe this should be a decorator rather than duplicate code.

Reply:
@dmbaturin Could you send an example of the code?

Reply:
> I believe this should be a decorator rather than duplicate code.

Done!